<launch>

    <node name="compressed_ro_raw_flip_with_param1" pkg="image_processing" type="compressed2raw.py" output="screen">
        <param name="sub_camera_topic" value="/camera1/color/image_raw/compressed"/>
        <param name="pub_camera_topic" value="/camera1/color/image_raw"/>
        <param name="flip" value="false"/>
    </node>

    <node name="compressed_ro_raw_flip_with_param2" pkg="image_processing" type="compressed2raw.py" output="screen">
        <param name="sub_camera_topic" value="/camera2/color/image_raw/compressed"/>
        <param name="pub_camera_topic" value="/camera2/color/image_raw"/>
        <param name="flip" value="false"/>
    </node>

    <node name="compressed_ro_raw_flip_with_param3" pkg="image_processing" type="compressed2raw.py" output="screen">
        <param name="sub_camera_topic" value="/camera3/color/image_raw/compressed"/>
        <param name="pub_camera_topic" value="/camera3/color/image_raw"/>
        <param name="flip" value="false"/>
    </node>

</launch>